site/routes/definitions.js

```diff
@@ -40,6 +40,13 @@ export const routeDefinitions : Array<RouteDefinition> = [
     stateToProps: ({ events }) => ({ events }),
   },
   {
+    title: event => event.title,
+    key: 'event',
+    route: 'about-us/events/{day}/{month}/{year}/{slug}',
```

Comment:
Would it be better have {year}/{month}/{day} ?
This way it makes more sense from a URL perspective. The year contains the months and then the days.
It's what usually see on blogs.